internal/provider/access_policies_data_source.go

```diff
@@ -73,8 +73,50 @@ func (d *accessPoliciesDataSource) Schema(_ context.Context, _ datasource.Schema
 							ElementType: types.StringType,
 						},
 						"credential_provider": schema.StringAttribute{
-							Description: "Credential Provider ID configured in the Access Policy.",
+							Description:        "Configured server workload of the access policy.",
```

Comment:
Should this description be for the Credential Provider, not the server workload?